src/gui/guiFormSpecMenu.cpp

```diff
@@ -353,7 +440,7 @@ void GUIFormSpecMenu::parseList(parserData* data, const std::string &element)
 		else
 			loc.deSerialize(location);
 
-		v2s32 pos = getElementBasePos(true, &v_pos);
```

Comment:
Is `pos += AbsoluteRect.UpperLeftCorner;` not necessary here?

Reply:
No, absolute positions aren't necessary anywhere anymore as every element is now a child and as such has the `AbsoluteRect` of its parent as offset (if parent is `this`, the offset will still be `AbsoluteRect.UpperLeftCorner`).